src/pages/solutions/for-students.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: We help students 
+quickLinks:
+  - name: Data
+    anchor: data
+  - name: Basemaps and Imagery
+    anchor: imagery
+  - name: Conferences and Events
+    anchor: conferences
+  - name: Jobs and Internships
+    anchor: jobs
+  - name: Metadata
+    anchor: metadata
+  - name: Citations
+    anchor: citations
+
+---
+
+import Layout from '../../layouts/FullWidth.astro';
+import Section from '../../components/page/Section.astro';
+import QuickLinks from '../../components/page/QuickLinks.astro';
+
+<Layout title={frontmatter.title}>
+  <QuickLinks links={frontmatter.quickLinks}/>
+  <Section stripe title="Data">
+
+    `Data! Data! Data! I can't make bricks without clay!` - Sherlock Holmes
+
+    The <a href="/products/sgid/">State Geospatial Information Datasource</a> has a large <a href="/products/sgid-index/">catalog</a> of freely-available data you can use in your classwork, projects, or personal tinkering. 
```

Comment:
Can we identify some popular datasets or examples where students use data to help make the connection?